bin/aapp_dr_runner.py

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 
-# Copyright (c) 2014-2018 PyTroll community
+# Copyright (c) 2014-2018, 2021 PyTroll community
```

Comment:
Should this be 2014-2018, 2021 or 2014-2021?  There were commits in 2019 and 2020 too.